test/pollfd.jl

```diff
@@ -133,10 +133,14 @@ end
     t = Timer(0.0, 0.1)
     finalizer(t, t -> a[] += 1)
     wait(t)
-    e = @elapsed for i = 1:5
+    # Measure time with a precision of ms
+    t1 = time()
+    for i = 1:5
         wait(t)
     end
-    @test 1.5 > e > 0.5
```

Comment:
Can we just do `@test 1.5 > e > 0.4`  with a comment about possible timing jitter?